console.py

```diff
@@ -17,7 +17,7 @@
 
 class HBNBCommand(cmd.Cmd):
 
-    """A HBNBCommand class"""
+    """A HBNBCommand class for the command interpreter."""
```

Comment:
I think it was fine as it was before, short and straight to the point, please undo